docs/development/local_setup.md

```diff
@@ -133,345 +130,3 @@ cd gardener
 ```
 
 > Note: Gardener is using Go modules and cloning the repository into `$GOPATH` is not a hard requirement. However it is still recommended to clone into `$GOPATH` because `k8s.io/code-generator` does not work yet outside of `$GOPATH` - [kubernetes/kubernetes#86753](https://github.com/kubernetes/kubernetes/issues/86753).
-
-## Start the Gardener
```

Comment:
Maybe we should have a section `Where to go from here?` that links back to our `getting_started_locally.md` guide.